packages/gazelle_cli/lib/commands/uninstall/uninstall_project.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:io';
+
+/// Uninstalls a Gazelle project.
+class UninstallProjectError {
+  /// The error message.
+  final String message;
+
+  /// Creates a [UninstallProjectError].
+  const UninstallProjectError() : message = "Failed to uninstall project!";
+}
+
+/// Uninstalls a Gazelle project.
+///
+/// Throws a [UninstallProjectError] if project does not exist.
+
+Future<String> uninstallProject(
+  String projectName, {
+  String? path,
+}) async {
+  final nameOption = projectName;
+  final pathOption = path;
+
+  final completePath =
+      pathOption != null ? "$pathOption/$nameOption" : nameOption;
+  if (!await Directory(completePath).exists()) {
+    throw UninstallProjectError();
+  }
+
+  await Directory(completePath).delete(recursive: true);
+
+  return completePath;
```

Comment:
I would make some changes here.
Firstly, I would try to load the project configuration with the
`loadProjectConfiguration` function available in the `lib/commons` folder.

Then, I would ask the user if it really wants to delete the project, to avoid any user error.

What do you think of this?

Reply:
I'll update the code and will do all the changes

Reply:
Thank you! Let me know when you'll be ready 🚀